src/snapred/backend/data/LocalDataService.py

```diff
@@ -69,16 +67,18 @@ def _createFileNotFoundError(msg, filename):
 class LocalDataService:
     instrumentConfig: "InstrumentConfig"
     verifyPaths: bool = True
+    _rolodex: Dict[Tuple[str, bool, IndexorType], Indexor]
 
-    # starting version number -- the first run printed
-    VERSION_START = Config["instrument.startingVersionNumber"]
     # conversion factor from microsecond/Angstrom to meters
     CONVERSION_FACTOR = Config["constants.m2cm"] * PhysicalConstants.h / PhysicalConstants.NeutronMass
 
     def __init__(self) -> None:
         self.verifyPaths = Config["localdataservice.config.verifypaths"]
         self.instrumentConfig = self.readInstrumentConfig()
         self.mantidSnapper = MantidSnapper(None, "Utensils")
+        self._rolodex = {}
```

Comment:
This is a cache of `Indexor` objects which are accessed using the `indexor` method.

Reply:
The reason was because the method arguments are different  from the key the indexor should associate with.  However, I think I can do this with two separate methods.